File: jars/src/data/scripts/dakka_ModPlugin.java
package data.scripts;

import com.fs.starfarer.api.BaseModPlugin;
import data.scripts.plugins.dakka_ProjectileGlowPlugin;

public class dakka_ModPlugin extends BaseModPlugin {
    @Override
    public void onApplicationLoad() throws Exception {
        dakka_ProjectileGlowPlugin.getGlowData();
    }
}

File: jars/src/data/scripts/dakka_ModPlugin.kt
package data.scripts

import com.fs.starfarer.api.BaseModPlugin
import data.scripts.plugins.dakka_ProjectileGlowPlugin.Companion.glowData

class dakka_ModPlugin : BaseModPlugin() {
    @Throws(Exception::class)
    override fun onApplicationLoad() {
        glowData
    }
}